src/functionalities/OutputNode.js

```diff
@@ -51,6 +51,16 @@ class OutputNode extends Functionality {
     }
   }
 
+  get source() {
+    const { sources, name } = this
+
+    if (sources.length > 1) {
+      throw new Error(`Output Node: ${name} has more than one connected fct. Output nodes may only have one source. Current sources: ${sources}`)
+    }
```

Comment:
shouldn't this be part of the connecting logic (and I think it already is iirc). The query should not be concerned with this kind of assertions I think.

Reply:
this is currently linked tot he connecting logic. This is just another failsafe. 
The reason I have it here is because for OutputNodes you have this special behavior (they can only ever have one connection). It was for this special behavior that I wanted to be super clear in case anyone comes across this down the line.